src/routes/pages/zh/manage-your-sats.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Manage your sats safely
```

Comment:
@CodyTseng looks like this one got missed.

Reply:
actually this page isn't show on the site so you can leave it for now. There is however, one new page "setup-zapping-wallet.md" that needs to be translated when you have time.

Reply:
Okay, I will translate it as soon as possible.

Reply:
@CodyTseng did you get a chance to translate this last page? If you don't have time for now, I'll probably merge this PR and you can add that last page in another PR when you have a moment.

Reply:
I have been working a lot recently, and I had a fever 🤒 yesterday. The translation may not be completed recently. I'll finish it on another PR when I have time.

Reply:
No worries at all! Take your time and I hope you feel better soon.